packages/core/docs/commercetools/composables/use-forgot-password.md

```diff
@@ -0,0 +1,98 @@
+# `useForgotPassword`
+
+## Features
+
+`useForgotPassword` composable can be used to:
+
+* generate reset password token
+* change user password using token
+
+## API
+
+- `reset` - function used to generate reset password token. When invoked, it requests data from the API and populates `token` property. This method accepts a single params object. The `params` has the following option:
+
+  - `params: IResetPasswordParams`
+
+  - `customQuery?: CustomQuery`
+
+```typescript
+interface IResetPasswordParams {
+    email: string;
+}
+
+type CustomQuery = {
+  customerCreatePasswordResetToken: string
+}
+```
+
+- `change` - function used to set new user password after reset. When invoked, it requests data from the API and populates `result` property. This method accepts a single params object. The `params` has the following option:
+
+  - `params: IChangePasswordParams`
+  
+  - `customQuery?: CustomQuery`
+
+```typescript
+interface IChangePasswordParams {
+    tokenValue: string;
+    newPassword: string;
+}
+
+type CustomQuery = {
+  customerResetPassword: string
+}
+```
+
+- `token: string` - reactive data string containing the created token.
+
+- `loading: boolean` - reactive object containing information about loading state of `change` and `reset` methods.
+
+- `error: UseForgotPasswordErrors` - reactive object containing the error message, if `change` or `reset` failed for any reason.
+
+```ts
+interface UseForgotPasswordErrors {
+  result: Error;
```

Comment:
As the two methods cannot be called in one step (you genereate reset token in one page and change password in another) there was no need for me to generate separate keys for errors in these methods. This results in simpler template (as we do not need to create separate conditionals for errors, we just have one) and in less logic because there is only one error. Do you think there should be separate errors for these methods?